### PR TITLE
Lexus RX 2016-2017

### PR DESCRIPTION
**Pull request by illumiN8i**
# Car support
This pull requests adds support for Lexus - RX - 2016-2017 - 350.

Updated version of @wocsor's PR #723 with fingerprint confirmed by discord user hkowald

This is an explorer link to a drive with the stock system enabled: https://my.comma.ai/cabana/?route=7b006d95b8798465%7C2020-01-06--16-38-50&exp=1609913706&sig=0zrNPVUxdQ%2FIM9jyuMYqfecWG6CJQAvph9MuPvNqh14%3D&max=-1&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F7b006d95b8798465%2Fd187f2dbbef05d857a679fc992fe6591_2020-01-06--16-38-50

This is an explorer link to a drive with openpilot system enabled: https://my.comma.ai/cabana/?route=7b006d95b8798465%7C2020-01-06--16-20-49&exp=1609913892&sig=svog6OHZBljIRYZXLhBvauNGpWJVaYARig3FuLAvhEA%3D&max=-1&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F7b006d95b8798465%2F4d73f824b132f2cc2bf88e228bc549e5_2020-01-06--16-20-49

Additional fingerprint for 2017 Lexus RX 350 from discord user slavka89:

Stock system enabled: https://my.comma.ai/cabana/?route=886fcd8408d570e9%7C2020-01-15--23-50-03&exp=1610670022&sig=Y8H4lf0W2E0xlB8nePpX%2Fpj3nWsFT6NP43AQBqeJnnc%3D&max=-1&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F886fcd8408d570e9%2Ff85f95e4b79e4f16fa5876ae751f0586_2020-01-15--23-50-03

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -258,6 +267,7 @@ class CAR:
   CAR.RAV4: dbc_dict('toyota_rav4_2017_pt_generated', 'toyota_adas'),
   CAR.PRIUS: dbc_dict('toyota_prius_2017_pt_generated', 'toyota_adas'),
   CAR.COROLLA: dbc_dict('toyota_corolla_2017_pt_generated', 'toyota_adas'),
+  CAR.LEXUS_RX: dbc_dict('LEXUS_RX_2016_pt_generated', 'toyota_adas'),
```
Comment (pd0wm) [inline selfdrive/car/toyota/values.py]: Does it work with the DBC name being uppercase?

Comment (illumiN8i) [reply selfdrive/car/toyota/values.py]: Oops, that was a find and replace mistake. Fixed

**Comment by geohot** [comment]
Can you add the new car route to the tests in selfdrive/test/test_car_models.py?

Then we have an internal script to copy the route to the CI bucket, and will merge this once tests pass.

**Comment by illumiN8i** [comment]
Added

**Comment by pd0wm** [comment]
That route is already gone :(

Can you collect one route with DSU plugged in and one without?

**Comment by geohot** [comment]
886fcd8408d570e9|2020-01-15--23-50-03 looks fine, can we use that?

**Comment by illumiN8i** [comment]
886fcd8408d570e9|2020-01-15--23-50-03 is a stock drive.  Will that work for CI?

I can contact the the car owners and ask them to collect new openpilot enabled routes with DSU connected and disconnected.

**Comment by illumiN8i** [comment]
Added new routes from discord user slavka89. @pd0wm can you upload them to the CI bucket quickly before they get deleted again?

I'm assumed 'enableDsu': True, means DSU disconnected and 'enableDsu': False, means DSU connected. If that's not correct let me know.

**Pull request merged**